Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,469 @@
+direction of the outgoing flow. The particles travel away from the nozzle at an angle and this increases the horizontal velocity component, and decreases the
+vertical velocity (the direction pointing out of the nozzle), which decreases the thrust of the system. Since cold gas propulsion is inherently inefficient, due
+to the fact that their is no chemcial reaction or heat applied to the system, maxmizing the thrust is imperitive.
+
+Last, but certainly not least, is the nozzle determination experience. Because of the flow conditions and dimensions, an analytical solution will not be sufficient. To accurately determine
+the nozzle size experiementation must happen. This data will be immensely helpful for designing the RCS nozzles and any other CGT nozzles with similar
+size magnitudes.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include 4 to 5 weeks of design and 4 to 5 weeks of data analysis. The data analysis will go into winter break because RIT facilites are not needed to analyze data. Once the design phase
+is complete, testing the system will be the primary concern of the team. The project will start off with a redesign discussion of the test stand, optmizing it for new conditions if necessary.
+The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+nozzle dimensions to create the polynomial curve approximation that will be an integral part of the rest of the research on the topic. Once that is complete, the
+testing and experimentation phase begins. This is the projected path for the investigation of the cold gas thruster.
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The deliverables of this project will be split into to two sections, physical and non-phsyical.
+
+\subsubsection{Physical Deliverables}
+\label{subsubsec: physical deliverables}
+
+The propulsion system will include a regulator,
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
+consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
+There will also be an arduino connected to a computer that will use Simulink for data acquisition. The arduino will be a conduit to relay the data from the load cell
```

Comment:
I would like to, but I do not know for certain that we can get one of those. Are they not expensive? Also, I would like David to handle this section because I know very little about this stuff. I am taking my first class on it this semester.